### ISSUE TITLE
Problem loading model?

### ISSUE THREAD

**Issue by logiclrd** [helix-toolkit/helix-toolkit#2188]
I want to display a model in a 3D viewport as part of my WPF application. When I add the model to my project and double-click it in Visual Studio, it displays as expected:

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/b3399c8f-920c-42fe-9b5b-c6d41f8d41b0)

I've never done this before, so I found a tutorial and followed it step by step, and when I finished all the steps and ran it, my window looked like this:

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/5130729b-a9c4-4be2-8dfb-b21235802d12)

I found myself wondering if this were a "white on a white background" issue, so I made the window blue, and sure enough, as I drag the camera around, _something_ is visible.

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/b8fa1c36-92ea-40b4-aa1f-df093ee318f4)

This is some sort of reflection off of the surface; as the camera moves, the weird little swatches of white move smoothly too. But, the surface of the mug itself doesn't seem to be rendered. I assumed I must be doing something wrong, since I'm brand new to this, so I found a sample online someone else had made of a simple OBJ file viewer, and when I fire that up, I get:

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/a9008761-78b0-4af2-b1f1-49d1278f1221)

Hmm, white-on-white again? I made this one also have a blue background, and:

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/bc6d26ec-1dab-4198-a9a3-0e4376cf6ac4)

Well, this looks familiar! So, given that the sample is ostensibly known to work, it seems I'm not doing anything wrong on the code side. And, Visual Studio loads and displays the model just fine, so it seems the model itself isn't broken. What's going wrong? Is there something I can do differently to make this work??

I'm placing the viewport like this:
```
        <Viewport3D Name="v3dViewport" Grid.ColumnSpan="2" Grid.RowSpan="2" MouseDown="v3dViewport_MouseDown" MouseMove="v3dViewport_MouseMove" MouseUp="v3dViewport_MouseUp" />
```

I'm loading the model like this:
```
			var reader = new ObjReader(Dispatcher);

			var model = reader.Read(@"B:\code\SubDesigner\Mug.obj");

			model.Children.Add(new AmbientLight(Color.FromRgb(190, 190, 190)));
			model.Children.Add(new DirectionalLight() { Direction = new Vector3D(3, -4, 5), Color = Colors.White });

			_visual = new ModelVisual3D();
			_visual.Content = model;
			_visual.Transform = new Transform3DGroup();

			v3dViewport.Camera =
				new PerspectiveCamera()
				{
					Position = new Point3D(0, 0, -20),
					LookDirection = new Vector3D(0, 0, 1),
					UpDirection = new Vector3D(0, 1, 0),
					NearPlaneDistance = 0,
					FarPlaneDistance = 50,
					FieldOfView = 45
				};

			v3dViewport.Children.Add(_visual);
			v3dViewport.Children.Add(new DefaultLights());
```

Am I doing something obviously wrong? Thanks!

**Comment by holance** [comment]
Please upload a sample project so we can help you to debug more easily.

**Comment by logiclrd** [comment]
I found out the problem :-) Bear in mind that I'm _super new_ to all this, so this is probably the first thing you'd have checked. But, the `.mtl` file referred to an external image file for its `map_Kd` (it came that way from the place I found it), and that reference was broken. When I fixed that, it started working properly. :-)

![image](https://github.com/helix-toolkit/helix-toolkit/assets/2349379/49f84f89-db69-41b6-91c6-0311b6fb91f7)

I've since figured out how to dynamically draw to the texture at runtime (which I need for this project). It looks like I'm off to the races. :-)
